cnd/src/config/settings.rs

```diff
@@ -489,4 +454,22 @@ mod tests {
                 lnd: Lnd::default(),
             })
     }
+
+    #[test]
+    fn error_on_http_url_for_lnd() {
```

Comment:
:thinking: I guess it would be just as valuable and more precise to test `fn check_url_lnd`.

Reply:
I would argue that testing the public API that is actually in use is more robust because it allows us to refactor the internals and still use the tests to make sure we didn't break anything :)